apps/storefront/src/app/[locale]/not-found.tsx

```diff
@@ -1,12 +1,101 @@
-export default function NotFound() {
+import Image from "next/image";
+import Link from "next/link";
+
+import { Footer } from "@/components/footer";
+import { Header } from "@/components/header";
+import { CACHE_TTL } from "@/config";
+import { getLocalePrefix } from "@/lib/server";
+import { getCurrentRegion } from "@/regions/server";
+import { cmsMenuService } from "@/services/cms";
+
+import { Navigation } from "./(main)/_components/navigation";
+
+export default async function NotFound() {
+  const [region, locale] = await Promise.all([
+    getCurrentRegion(),
+    getLocalePrefix(),
+  ]);
+
+  const resultMenu = await cmsMenuService.menuGet({
+    channel: region.market.channel,
+    languageCode: region.language.code,
+    locale,
+    slug: "navbar",
+    options: {
+      next: {
+        tags: ["CMS:navbar"],
+        revalidate: CACHE_TTL.cms,
+      },
+    },
+  });
+
   return (
-    <html lang="en">
-      <body>
-        <div className="min-h-screen w-full">
-          <h2 className="text-center text-8xl">404</h2>
-          <h2 className="text-center text-2xl">Not found</h2>
+    <>
+      <div className="bg-background sticky top-0 isolate z-50 py-4 md:pb-0">
+        <Header />
+        <Navigation menu={resultMenu.data?.menu} />
+      </div>
+      <main className="bg-background container sm:mb-16 sm:mt-20">
+        <div className="xs:items-center flex w-full flex-wrap-reverse justify-between bg-transparent pl-2 shadow-none sm:flex-nowrap">
+          <section
+            className="text-black xl:max-w-[492px] dark:text-white"
+            aria-labelledby="not-found-heading"
+          >
+            <h1
+              id="not-found-heading"
+              className="mb-8 hidden text-2xl font-medium leading-none sm:block sm:text-3xl md:text-3xl xl:text-5xl"
+            >
+              Oops! Page Not Found
+            </h1>
+            <p className="mb-6 text-base leading-7 tracking-normal">
+              Looks like you’ve hit a dead end, but don’t worry, we’ve got a
+              <Link
+                href="/"
+                className="font-semibold underline underline-offset-2"
+              >
+                {" "}
+                map!
+              </Link>
+            </p>
+            <p className="mb-2 text-base">
+              Or, if you’re feeling adventurous...
+            </p>
+            <p className="max-w-[467px] text-base">
+              <Link
+                href="/"
+                className="font-medium underline underline-offset-2"
+              >
+                Click here
+              </Link>
+              <span>
+                {" "}
+                to discover a random page – who knows where you’ll end up?
+              </span>
+            </p>
+          </section>
+
+          <div className="relative aspect-square w-full sm:ml-auto sm:aspect-[4/3] sm:max-w-[740px]">
+            <Image
+              src="/cube.png"
+              alt="Cube Logo"
```

Comment:
Rename the asset to nimara-storefront-cube.png or something more meaningful. Since it's a static file put it in the assets directory and import it here:
```suggestion
              src={nimaraCubeLogo}
              alt="Nimara Storefront logo"
```